src/index.html

```diff
@@ -1,13 +1,508 @@
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">11:00 — 19:00</span> субота
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">11:00 — 19:00</span> неділя
+                    </li>
+                  </ul>
+                </div>
+                <p class="schedule__weekend">
+                  <span class="schedule__weekend--strong">Вихідні:</span> понеділок, вівторок<br> Каса припиняє роботу за 1 годину до закриття музею
+                </p>
+              </div>
+            </div>
+            <div class="logo top-bar__logo">
+              <a href="#home">na<br>mu</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="header__overlay"></div>
+    </header>
+    <main class="main">
+      <section class="section">
+        <div class="section__container">
+          <div class="hero js-anim-items">
+            <div class="hero__overlay">
+              <img
+                src="./images/hero/hero-mobile.jpg"
+                srcset="./images/hero/hero-mobile.jpg 320w, ./images/hero/hero-1x.jpg 400w, ./images/hero/hero-2x.jpg 750w"
+                alt="Фото музею"
+                class="hero__image"
+                sizes="(max-width: 767px) 320px, (max-width: 1279px) 400px, (min-width: 1280px) 750px"
+              >
+            </div>
+            <div class="hero__info">
+              <h1 class="hero__title anim-show js-anim-no-hide">Художній музей</h1>
+              <a href="#describe" class="button hero__button anim-show js-anim-no-hide">
+                Квитки
+                <div class="button__icons">
+                  <svg
+                  class="icon button__icon"
+                  width="24"
+                  height="24"
+                  aria-label="Квитки"
+                  >
+                    <use href="./images/icons/sprite.svg#arrow-button"></use>
+                  </svg>
+                </div>
+              </a>
+              <a href="#events" class="hero__events">Події</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="section" id="events">
+        <div class="events js-anim-items">
+          <div class="events__content">
+            <h2 class="events__title anim-show js-anim-no-hide">Актуальні події</h2>
+            <ul class="events__list">
+              <li class="events__item">
+                <article class="card events__card events__card--padding js-anim-items">
+                  <div class="card__images card__images-first anim-show">
+                    <img
+                      src="./images/events/goReturn-mobile.jpg"
+                      srcset="./images/events/goReturn-mobile.jpg 320w, ./images/events/goReturn-1x.jpg 400w, ./images/events/goReturn-2x.jpg 750w"
+                      alt="Йду і повертаюсь"
```

Comment:
Try to improve alt description for events. Just think how to better understand image if you don't see it

Something like `Event name: Name` or `Event - Name`